src/multisig/keygen/secret_key_share.rs

```diff
@@ -46,19 +53,27 @@ impl GroupPublicInfo {
         self.threshold
     }
 
-    pub fn pubkey_bytes(&self) -> BytesVec {
-        todo!()
-    }
-
-    pub fn all_shares_bytes(&self) -> TofnResult<BytesVec> {
-        // encode(&self.all_shares)
-        todo!()
-    }
-
     pub fn all_verifying_keys(&self) -> &VecMap<KeygenShareId, k256_serde::ProjectivePoint> {
         &self.all_verifying_keys
     }
 
+    /// tofnd can send this data through grpc
+    pub fn all_verifying_keys_bytes(&self) -> TofnResult<Vec<KeygenShare>> {
```

Comment:
Calling this pubkey might be more readable, especially now that we're not using the `VerifyingKey` struct?

Reply:
Oh, yeah, I meant `all_pubkeys_bytes` (or if we want to deviate a bit, `all_encoded_pubkeys`). Similar for the method above this?

Reply:
Ok just for you I'm switching to `pubkey`, `encoded_pubkey`, `all_pubkeys`, `all_encoded_pubkeys`.  @sdaveas this means I've renamed GG20 `pubkey_bytes` -> `encoded_pubkey` which is a breaking change for tofnd.

Honestly I'm starting to prefer `verifying_key` to `pubkey` because `pubkey` is ambiguous.  (Is it a pubkey for encryption?  Signature?)  But let's leave it as-is for now because I'm getting sick of renaming things 😛

Reply:
Haha, fair enough